libs/ett/auth/data-access-security/src/lib/authorization/ett-authorization.interceptor.ts

```diff
@@ -38,7 +38,6 @@ export class EttAuthorizationInterceptor implements HttpInterceptor {
     request: HttpRequest<unknown>,
```

Comment:
question/suggestion:

Do we know the type given to the "intercept" method -> "Observable<HttpEvent<**unknown**>>"?
Do we know the type of error for the parameter to "#is403ForbiddenResponse"?

Reply:
No, potentially any error type can be passed to an interceptor, but we're asserting the type for the error that this interceptor is handling (`HttpErrorReponse` with HTTP status code `403`). We're just handling this specific error by performing a side effect but also delegating it to other interceptors and error handlers, hence the `tap` operator in line 42.